data/builtInFeatures.js

```diff
@@ -0,0 +1,134 @@
+  "es6.typed.uint16-array": "typed arrays / Uint16Array",
+  "es6.typed.int32-array": "typed arrays / Int32Array",
+  "es6.typed.uint32-array": "typed arrays / Uint32Array",
+  "es6.typed.float32-array": "typed arrays / Float32Array",
+  "es6.typed.float64-array": "typed arrays / Float64Array",
+
+  "es6.map": "Map",
+  "es6.set": "Set",
+  "es6.weak-map": "WeakMap",
+  "es6.weak-set": "WeakSet",
+  // proxy not implemented
+
+  "es6.reflect.apply": "Reflect / Reflect.apply",
+  "es6.reflect.construct": "Reflect / Reflect.construct",
+  "es6.reflect.define-property": "Reflect / Reflect.defineProperty",
+  "es6.reflect.delete-property": "Reflect / Reflect.deleteProperty",
+  "es6.reflect.get": "Reflect / Reflect.get",
+  "es6.reflect.get-own-property-descriptor": "Reflect / Reflect.getOwnPropertyDescriptor",
+  "es6.reflect.get-prototype-of": "Reflect / Reflect.getPrototypeOf",
+  "es6.reflect.has": "Reflect / Reflect.has",
+  "es6.reflect.is-extensible": "Reflect / Reflect.isExtensible",
+  "es6.reflect.own-keys": "Reflect / Reflect.ownKeys",
+  "es6.reflect.prevent-extensions": "Reflect / Reflect.preventExtensions",
+  "es6.reflect.set": "Reflect / Reflect.set",
+  "es6.reflect.set-prototype-of": "Reflect / Reflect.setPrototypeOf",
+
+  "es6.promise": "Promise",
+
+  "es6.symbol": {
+    features: [
+      "Symbol",
+      "well-known symbols / Symbol.hasInstance",
+      "well-known symbols / Symbol.isConcatSpreadable",
+      "well-known symbols / Symbol.iterator",
+      "well-known symbols / Symbol.match",
+      "well-known symbols / Symbol.replace",
+      "well-known symbols / Symbol.search",
+      "well-known symbols / Symbol.species",
+      "well-known symbols / Symbol.split",
+      "well-known symbols / Symbol.toPrimitive",
+      "well-known symbols / Symbol.toStringTag",
+      "well-known symbols / Symbol.unscopables",
+    ]
+  },
+
+  "es6.object.assign": "Object static methods / Object.assign",
+  "es6.object.is": "Object static methods / Object.is",
+  "es6.object.get-own-property-symbols": "Object static methods / Object.getOwnPropertySymbols",
+  "es6.object.set-prototype-of": "Object static methods / Object.setPrototypeOf",
+
+  "es6.function.name": 'function "name" property',
+
+  "es6.string.raw": "String static methods / String.raw",
+  "es6.string.from-code-point": "String static methods / String.fromCodePoint",
+
+  "es6.string.code-point-at": "String.prototype methods / String.prototype.codePointAt",
+  // "String.prototype methods / String.prototype.normalize" not implemented
+  "es6.string.repeat": "String.prototype methods / String.prototype.repeat",
+  "es6.string.starts-with": "String.prototype methods / String.prototype.startsWith",
+  "es6.string.ends-with": "String.prototype methods / String.prototype.endsWith",
+  "es6.string.includes": "String.prototype methods / String.prototype.includes",
+
+  "es6.regexp.flags": "RegExp.prototype properties / RegExp.prototype.flags",
```

Comment:
You mean for http://babeljs.io/docs/plugins/transform-es2015-unicode-regex/#installation?

(we need an example there)